templates/compose/signoz.yaml

```
@@ -0,0 +1,627 @@
+                <replace>\1(???)</replace>
+              </rule>
+            </query_masking_rules>
+            <send_crash_reports>
+              <enabled>false</enabled>
+              <anonymize>false</anonymize>
+              <endpoint>https://6f33034cfe684dd7a3ab9875e57b1c8d@o388870.ingest.sentry.io/5226277</endpoint>
+            </send_crash_reports>
+            <merge_tree_metadata_cache>
+              <lru_cache_size>268435456</lru_cache_size>
+              <continue_if_corrupted>true</continue_if_corrupted>
+            </merge_tree_metadata_cache>
+            <user_directories>
+              <users_xml>
+                  <!-- Path to configuration file with predefined users. -->
+                  <path>users.xml</path>
+              </users_xml>
+              <local_directory>
+                  <!-- Path to folder where users created by SQL commands are stored. -->
+                  <path>/var/lib/clickhouse/access/</path>
+              </local_directory>
+            </user_directories>
+            <default_profile>default</default_profile>
+              <distributed_ddl>
+                  <!-- Path in ZooKeeper to queue with DDL queries -->
+                  <path>/clickhouse/task_queue/ddl</path>
+              </distributed_ddl>
+          </clickhouse>
+
+  signoz:
+    image: signoz/signoz:latest
+    depends_on:
+      clickhouse:
+        condition: service_healthy
+      schema-migrator-sync:
+        condition: service_completed_successfully
+    restart: unless-stopped
+    logging:
+      options:
+        max-size: 50m
+        max-file: "3"
+    command:
+      - --config=/root/config/prometheus.yml
+    volumes:
+      - type: bind
+        source: ./prometheus.yml
+        target: /root/config/prometheus.yml
+        content: |
+          # my global config
+          global:
+            scrape_interval:     5s # Set the scrape interval to every 15 seconds. Default is every 1 minute.
+            evaluation_interval: 15s # Evaluate rules every 15 seconds. The default is every 1 minute.
+            # scrape_timeout is set to the global default (10s).
+
+          # Alertmanager configuration
+          alerting:
+            alertmanagers:
+            - static_configs:
+              - targets:
+                - alertmanager:9093
+
+          # Load rules once and periodically evaluate them according to the global 'evaluation_interval'.
+          rule_files: []
+            # - "first_rules.yml"
+            # - "second_rules.yml"
+            # - 'alerts.yml'
+
+          # A scrape configuration containing exactly one endpoint to scrape:
+          # Here it's Prometheus itself.
+          scrape_configs: []
+
+          remote_read:
+            - url: tcp://clickhouse:9000/signoz_metrics
+      - type: volume
+        source: sqlite
+        target: /var/lib/signoz/
+    environment:
+      - SERVICE_FQDN_SIGNOZ_8080
```

services now need to use the `SERVICE_URL` variable instead of `SERVICE_FQDN` since the latest update from a few days ago.
```suggestion
      - SERVICE_URL_SIGNOZ_8080
```